baremaps-server/src/main/java/org/apache/baremaps/server/TileJSON/Field.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.baremaps.server.TileJSON;
+
+public class Field {
```

Comment:
It is new in tilejson-spec 3.0.0 https://github.com/mapbox/tilejson-spec/tree/master/3.0.0#332-fields 
```
An object whose keys and values are the names and descriptions of attributes available in this layer. Each value (description) MUST be a string that describes the underlying data. If no fields are present, the fields key MUST be an empty object.
```

It is empty but the class purpose would be to pick-up all the fields in a Java Map in a class attribute called `additionalProperties` (as per naming in json schema of the specification).

To be discuss what as part of the points is to which extent to add support for the tilejson-spec in this change.

Reply:
Interesting, thank you for the clarification. I wonder if using an Object instead of a Field would give the flexibility to put a Map in this attribute. What do you think? Both alternatives are fine for me.

```java
@JsonProperty("fields")
@Valid
@NotNull Field fields,
```